ChangeLog

```diff
@@ -2,7 +2,32 @@ DOMjudge Programming Contest Judging System
 
 Version 8.2.0 - DEV
 -------------------
- - Add support for Rust
+ - Various bugfixes & UI improvements.
+ - Add support for Rust as submission language.
+ - Add teamphotos to scoreboard zip.
+ - Add jury option to to download ZIP with all problem samples, attachments
+ and problem statement.
```

Comment:
Nitpick: I think it's better to indent this to the level of the start of the text on the previous line. See also changelogs below.

Reply:
Fixed for all lines.